modules/book.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+import {
+  container, header, form, titletext, contactpage, listTab, addingTab, contactTab,
+} from '../index.js';
+import { error } from './error.js';
+import { inputSave } from './localstorage.js';
+// import {bookarticles,directive,bookMessage} from './directive.js';
```

Comment:
- [ ] Please remove the commented code. The commented code makes your code messy and may make your code hard to maintain 👍.